CoopersHill/finance-SPA
# New user account

updated userAccount branch. has the commits from today and yesterday

I created an account management page. the description is a link to the transactions page with ownerid as the id parameter on that page.

im fairly confident in its functioning.

TEST INSTRUCTIONS
make sure you pull down the user account branch.

check if you have arn installed by running
yarn --v
in the terminal
once yarn is confirmed,
2.

navigate to the financespa folder, go to the terminal and run the follwing command
yarn start

make sure youre on the user account branch

the default page is the account management page.
let me know if you have issues running the project
